src/command_modules/azure-cli-policyinsights/azure/cli/command_modules/policyinsights/_validators.py

```diff
@@ -14,3 +14,15 @@ def validate_resource(cmd, namespace):  # pylint: disable=unused-argument
                 raise CLIError('--namespace is required if --resource is not a resource ID.')
             if not namespace.resource_type:
                 raise CLIError('--resource-type is required if --resource is not a resource ID.')
+
+
+def validate_resourceScopeRequired(cmd, namespace):
```

Comment:
Would `validate_resource` be better? Either way "resourceScopeRequired" has to be split to be snake style

Reply:
@yugangw-msft  Changed to `validate_resource_required`